cmake/testing.cmake

```diff
@@ -0,0 +1,214 @@
+function(mayaUsd_get_unittest_target unittest_target unittest_basename)
```

Comment:
This is a minor thing and I am Ok if you don't want to change:
I think `test.cmake` or ` unittest.cmake` are better names. The `-ing` form is a bit weird to me

Reply:
Sure, I don't really have a preference. I'll go with test.cmake

Reply:
Cool. test.cmake is fine.

Reply:
https://github.com/Autodesk/maya-usd/pull/242/commits/53f72ecf0431c641b04fc6d171fb3519564bb200